wrapper/pre_copy.py

```diff
@@ -690,12 +707,22 @@ def __repr__(self):
 
         return ETree.tostring(tree)
 
-    def get_xml(self):
+    def _get_xml(self):
         xmlfile = os.path.join(self._tmp_dir.name, 'vm.xml')
         with open(xmlfile, 'wb') as f:
             f.write(self._fix_disks(self.vmware.get_domxml()))
         return xmlfile
 
+    def prepare_command(self, v2v_env, v2v_args):
+        v2v_env['LIBGUESTFS_CACHEDIR'] = self._tmp_dir.name
```

Comment:
We place larger files in `/var/tmp` so you should make sure that `_tmp_dir` is properly initialized. Also don't ignore `LIBGUESTFS_CACHEDIR` if it is already defined in the environment.

Reply:
Covered by #46